1904-10-31.xml

```diff
@@ -0,0 +1,2512 @@
+                        kind that has ever been held here. It was so much larger than last year's
+                        that there was no room for the cut flowers in the large ball-room of the
+                        casino, and they and some of the pot plants had to be accommodated in the
+                        reading-room. Great credit is due to Lady Blomfield, Mr. Rolo, and Mme C.
+                        Debban, who, although living in the heart of Alexandria has never seen two
+                        such beautiful boxes of chrysanthemums as thise sent by Mr. Rolo, and
+                        amongst his collection was the best chrysanthemum in the show. The feature
+                        of the large groups were the masses of white bloom. All the groups showed
+                        much better taste than in the earlier exhibitions. The flowers were arranged
+                        by masses of colours, as they are in Europe, and there was not that mixture
+                        of colour which was far from artistic in the earlier shows. The Municipality
+                        showed some very fine blooms, but theywere not for competition. The show was
+                        well attended on both days. The following is the prize-list:– </p>
+                    <p>
+                        <list type="ordered">
+                            <item>Group of Chrysanthemums. (Minimum 200nplants).– Silver Medal,
+                                Prince Omar Pasha Toussoun; Bronze Medal M. Zervudachi; Highly
+                                Commended, Mme Debbane.</item>
+                            <item>Group of Chrysanthemums (75 to 100 plants).– 1st, Lady Blomfield;
+                                2nd Judge Sandars.</item>
+                            <item>18 Chrysanthemums, cultivated for large flowers (6 varieties at
+                                least).– 1st, Mrs. A. Borchgrevink; 2nd Prince Omar Tousoun.</item>
+                            <item>6 Chrysanthemums, cultivated for large flowers.– 1st, Prince Omar
+                                Toussoun; 2nd, Mme Ch. Debbane.</item>
+                            <item>Chrysanthemums, each plant to carry at least 15 flowers.– 1st, M.
+                                Zervudachi; 2nd, Prince Omar Toussoun.</item>
+                            <item>24 Chrysanthemums (12 varieties at least). Best chrysanthemum in
+                                the show to be selected by the judges from the boxes exhibited.–
+                                Silver Medal, Prince Omar Toussoun; Bronze Medal, M.
+                                Zervudachi.</item>
+                        </list>
+                    </p>
+                </div>
+                <cb n="5"/>
+                <div type="item">
+                    <p>10. 12 Chrysanthemums (mixed).– 1st, M. Jacques Rolo; 2nd, Prince Omar
```

Comment:
except that the list gets interrupted. no need for paragraph and div to break up; just drop the <cb/> where it's needed in the middle of the list.